Meter.c

```diff
@@ -121,8 +121,8 @@ void Meter_setMode(Meter* this, int modeIndex) {
       }
    } else {
       assert(modeIndex >= 1);
-      free(this->drawData);
-      this->drawData = NULL;
+      free(this->drawData.values);
+      this->drawData.values = NULL;
```

Comment:
Null pointer dereference is possible when you change the meter mode (style) to anything else and then back to graph.
You should set `this->drawData.nValues` to `0`. (I discovered this when working on my version of the patch (#1027))

Reply:
Good point; now fixed.